### ISSUE TITLE
Is the model structure different from the original paper?

### ISSUE THREAD

**Issue by ys198918** [seathiefwang/MGN-pytorch#8]
In your code, you use the global feature with 256d to do classification, and the paper said they use global features before reduction which is 2048d. Is that a mistake? or you did that for a reason?And by the way, in your code , you use adam with lr 0.0002 which is also different from the paper. Can you tell me why,? In my attemption , it seems like your setting is better than what the paper says.

**Comment by whut2962575697** [comment]
Hello, I found the same problem. Have you tried to use the 2048-dimensional feature before dimension reduction to classify ? Why use the settings in the code, the cross entropy loss is difficult to converge when training

**Comment by muzishen** [comment]
Hello, l found the same problems.  Have you tried to use the BNNeck in the global feature.  2048-d fed to triplet and after BN fed to softmax.
